epistrains/population.py

```diff
@@ -1,3 +1,21 @@
 class Population:
-    def __init__(self):
-        pass
+    """
+    Population based parameters for the construction of birth rate and death rate terms
+    Parameters
+    ----------
+    size: int
+        population size
+    death: float
+        constant death rate
+    a: float
```

Comment:
I think it makes sense to pass a function of population size when you instantiate the class (i.e. `Population(..., birth_func=lambda p: p**2)`)

Reply:
We could also have it default to the "bacterial" function

Reply:
Since the bacterial birth rate is dependent on the current population size I have created a separate function to be called from within the Solver class.